packages/components/src/components/password-meter/password-meter.jsx

```diff
@@ -1,28 +1,44 @@
 import classNames from 'classnames';
 import React from 'react';
-import zxcvbn from '@contentpass/zxcvbn';
 import PropTypes from 'prop-types';
 import Field from '../field';
+import Loading from '../loading';
 
 const PasswordMeter = ({ children, has_error, input }) => {
-    // 0 - 4 Score for password strength
-    const { score, feedback } = zxcvbn(input);
-    const width_scale = has_error && input.length ? 0.25 : 0.25 * (input.length && score < 1 ? 1 : score);
+    const zxcvbn = React.useRef();
+
+    const [is_loading, setLoading] = React.useState(true);
+    const [score, setScore] = React.useState(undefined);
```

Comment:
```suggestion
    const [score, setScore] = React.useState();
```

Reply:
`undefined` is the default value. It's not needed to set it explicitly

Reply:
Good point Bala :+1: